### PR TITLE
Cleared a wrong comment regarding path repository support

### PR DESCRIPTION
**Pull request by barbazul**
## Purpose of this pull request

Clarify support of the Path repository type

## Affected DevDocs pages

👉 [The composer.json file](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/build/composer-integration.html) 👈

### CODE REVIEW

File: src/guides/v2.3/extension-dev-guide/build/composer-integration.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Composer reads a `composer.json` file in Magento's root directory to download th
 We recommend you include `composer.json` in your component's root directory even if you do not intend to distribute it to other merchants using Magento.
 
  {:.bs-callout-info}
-Magento does not support the [`path`][3] repository.
+Magento does not support the [`path`][3] repository pointing to a folder outside of Magento root.
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/build/composer-integration.md:15]:
```suggestion
Magento does not support the [`path`][3] repository pointing to a folder outside of the Magento root.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also add the page URL to the PR description

**Comment by barbazul** [comment]
> Please also add the page URL to the PR description

It's already there. What am I missing?

**Comment by barbazul** [comment]
No pending requested changes

**Comment by barbazul** [comment]
@BarnyShergold

**Review by BarnyShergold — APPROVED** [review]
Apologies for the delay

**Review by jeff-matthews — COMMENTED** [review]
Thanks for the contribution @barbazul! We recently moved the 2.4 version of this page to another site and GitHub repo. This change will only affect version 2.3. Would you be willing to open a PR in the new repo with this change?

- **site:** https://developer.adobe.com/commerce/php/development/build/composer-integration/
- **repo:** https://github.com/AdobeDocs/commerce-php/blob/main/src/pages/development/build/composer-integration.md

**Comment by barbazul** [comment]
> Thanks for the contribution @barbazul! We recently moved the 2.4 version of this page to another site and GitHub repo. This change will only affect version 2.3. Would you be willing to open a PR in the new repo with this change?
> 
>     * **site:** https://developer.adobe.com/commerce/php/development/build/composer-integration/
> 
>     * **repo:** https://github.com/AdobeDocs/commerce-php/blob/main/src/pages/development/build/composer-integration.md

Here  you go: https://github.com/AdobeDocs/commerce-php/pull/22

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**